public/index.html

```diff
@@ -433,27 +433,25 @@ <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
 		<div class="container-fluid">
 			<div class="well row attributesWrapper" style="margin-top: 1%;">
 				<div class="col-md-3 wsdlIcon" >
-					<img src="/images/odu-sig-noidea-fullcolor.png" alt="ODULogo">
 					<img src="/images/glider-trans.png" alt="ODUWDL">
 				</div>
 				<div class="col-md-6" >
-					<p> Created by the <a href ="https://ws-dl.cs.odu.edu/">Web Science and Digital Libraries Research Group</a> at <a href ="https://cs.odu.edu/">Old Dominion University</a>. </p>
-					<p>Funded by
-						<a href="https://securegrants.neh.gov/publicquery/main.aspx?f=1&gn=HAA-256368-17"> NEH/IMLS Digital Humanities Advancement Grant HAA-256368-17</a>
-					</p>
+					<p style="padding-top: 10px;"> Created by the <a href ="https://ws-dl.cs.odu.edu/">Web Science and Digital Libraries Research Group <a href="https://twitter.com/WebSciDL?ref_src=twsrc%5Etfw">(@WebSciDL)</a></a> at <a href ="https://cs.odu.edu/">Old Dominion University</a>. </p>

```

Comment:
Anchor tags are not supposed to be nested and the query parameter in the Twitter link is unnecessary.

```suggestion
					<p style="padding-top: 10px;">Created by the <a href ="https://ws-dl.cs.odu.edu/">Web Science and Digital Libraries Research Group</a> (<a href="https://twitter.com/WebSciDL">@WebSciDL</a>) at <a href ="https://cs.odu.edu/">Old Dominion University</a>.</p>
```